src/terminal/input/terminalInput.cpp

```diff
@@ -507,16 +548,15 @@ void TerminalInput::_SendNullInputSequence(const DWORD dwControlKeyState) const
     }
 }
 
-void TerminalInput::_SendInputSequence(_In_ PCWSTR const pwszSequence) const
+void TerminalInput::_SendInputSequence(_In_ const std::wstring_view pwszSequence) const

```

Comment:
Urgh. Technicality. A `std::wstring_view` isn't guaranteed to be null terminated, so the `pwsz` prefix isn't valid. It would be `pws` to imply that it isn't necessarily z'd. 

However, if you're going to rename it, you may as well just name it `sequence` because we're slowly removing hungarian notation.

Reply:
Looks like you did this in the latest, so I'll resolve it.